2022/foss.html

```diff
@@ -188,7 +188,7 @@ <h1>FOSS & Diversity Communities</h1>
                                         <div class="row alternative-row">
                                             <div class="col-lg-12 col-sm-12 col-xs-12 site-content-left">
                                                 {% for entry in site.data.communities %}
-                                            <div class="entry entry-{{ entry.company.level }} col-md-6">
+                                            <div class="entry entry-{{ entry.company.level }} col-md-5">
```

Comment:
`level` property does not exist in `communities.yml`, seems save to delete

Reply:
You are right. Deleted that css style.